filebeat/module/elasticsearch/server/manifest.yml

```diff
@@ -4,10 +4,13 @@ var:
   - name: paths
     default:
       - /var/log/elasticsearch/*.log
+      - /var/log/elasticsearch/*_server.json
     os.darwin:
       - /usr/local/var/lib/elasticsearch/*.log
+      - /usr/local/var/lib/elasticsearch/*_server.json
```

Comment:
A quick question about these paths. For the unstructured logs it looks like we're grabbing `*.log` which I assume would also include the slow logs but then for the structured logs it looks like they're explicitly breaking the ES log out to be be suffixed with `*_server.json`. So it would appear from this that for unstructured logs we're slurping up more log files that we are for structured logs, no? Is this by intention?

Reply:
Good question. There's a couple pieces here that need to be tied together to see the full picture:

1. These variables here in the `manifest.yml` are just variables. As in, they don't (yet) determine any configuration.

2. The actual configuration is built from here: https://github.com/elastic/beats/blob/7777c1d8593b2e714468f9f831a7288af09141fb/filebeat/module/elasticsearch/server/config/log.yml#L1-L15

   As you can see, the `path` variable from (1) are used there but also notice the `exclude_files` setting. This is how we make sure we only slurp up structured logs.

Reply:
Gotcha. Makes perfect sense. Thanks!